app/models/govbox/api_connection_with_obo_support.rb

```diff
@@ -27,6 +27,6 @@ def destroy_with_box?
 
   def validate_box(box)
     box.errors.add(:settings_obo, :not_allowed) if obo.present?
-    box.errors.add(:settings_obo, :invalid) if boxes.where.not(id: box.id).where("settings ->> 'obo' = ?", box.settings_obo).exists?
+    box.errors.add(:settings_obo, :invalid) if boxes.where.not(id: box.id).where("settings @> ?", { obo: box.settings_obo }.to_json).exists?
```

Comment:
Tuto query preco menime? Ci je to kvoli tomu nil?

Reply:
Ano